Fitbit.Portable/FitbitClientExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Fitbit.Models;
+
+namespace Fitbit.Api.Portable
+{
+    internal static class FitbitClientExtensions
```

Comment:
@WestDiscGolf What do you think about renaming this to `FitbitClientHelper` to avoid confusion with a possible extension methods class for Fitbitclient.